pkg/dd/dashboard.go

```diff
@@ -1,14 +1,52 @@
 package dd
 
 import (
+	"fmt"
+
+	aw "github.com/deanishe/awgo"
 	"gopkg.in/zorkian/go-datadog-api.v2"
 )
 
-func QueryDashboard(client *datadog.Client) ([]datadog.DashboardLite, error) {
-	dashboards, err := client.GetDashboards()
-	if err != nil {
-		return nil, err
+// Dashobard is workflow structure of the dashboard.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [comment on exported type Dashboard should be of the form "Dashboard ..." (with optional leading article)](https://app.codacy.com/manual/nekottyo/alfred-datadog-workflow/pullRequest?prid=5421329)